<!-- -->
<launch>

	<arg name="detection_range" default="3.0" /> <!-- meter -->
	<arg name="stop_distance" default="5.0" /> <!-- meter -->

	<!-- rosrun lane_follower pure_pursuit _stop_distance:=0 _detection_range:=0 -->
	<node pkg="lane_follower" type="twist_filter" name="twist_filter" output="screen">
		<param name="stop_distance" value="$(arg stop_distance)" />
		<param name="detection_range" value="$(arg detection_range)" />
	</node>

</launch>
